# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1244182606 -7200
#      Fri Jun 05 08:16:46 2009 +0200
# Branch release-3-2-x
# Node ID de6919e0c0f3ef2e320e183112bbb4b759c4a7a5
# Parent  11767020f9664da81797c36dfc168305679ff92a
Added tag release-3-2-0 for changeset 530dc15eb653

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -38,8 +38,10 @@ 89d546610556ee47ec634ab8c3e441e1db248873
 274d9642ac1032fcc4996a3cac80cfe5fd20f8a4 release-3-0-0
 af10baa6391596c3713385d7b6c66c93ee935e87 ss-3-1-50
 a5d1e27ee1f4c3fd66c489e83fdb40f92bf1bf43 ss-3-1-51
 739141cde75a22b9ac9b5b327ea30c61fd357748 ss-3-1-52
 bd1b1fe9c6e952202bf5878cd042a168f6df6ddd ss-3-1-53
 3c3cbe8f18e03c73cc34494c8037d8a45a30bf7d ss-3-1-54
 5e276a0b999747b72af34e72eb63f64640da6399 ss-3-1-55
 a1d20052517a950033896f5a876de8f3ee520138 release-3-2-0
+a1d20052517a950033896f5a876de8f3ee520138 release-3-2-0
+530dc15eb653d538aaab0a139093354fbb6f0c68 release-3-2-0
